### ISSUE TITLE
Pages in a multi-page view can not be loaded lazily due to error

### ISSUE THREAD

**Issue by pvnhome** [primefaces/primefaces#1604]
Go to http://www.primefaces.org/showcase/mobile/lazyload.xhtml

Click "Go to Page 2" link. Nothing happening and you can see "TypeError: $(...).page is not a function" in the browser log.

**Comment by tandraschko** [comment]
Feel free to provide a PR

**Comment by mertsincan** [comment]
Please use h:form;

```
<pm:page id="main">
     <pm:header title="Lazy Load"></pm:header>

         <pm:content>
              <h:form>
                   <p>...</p>
                   <p:link outcome="pm:lazy" value="Go to Page 2" />
               </h:form>
....
```

I updated our demo page; https://github.com/primefaces/showcase/blob/ef1e044b06afb14fa8e8dd83e3d98b0e0b6b7077/src/main/webapp/mobile/lazyload.xhtml

**Comment by tandraschko** [comment]
PrimeFaces Mobile will be deprecated  in 6.2 and removed in 6.3 ( see https://www.primefaces.org/primefaces-6-2-roadmap/)
Therefor i will close this issue as won't fix. If you will provide a PR to fix this issue until 6.2 RC1, i can merge it.
